packages/plugin/src/theia.d.ts

```diff
@@ -2652,7 +2652,7 @@ export module '@theia/plugin' {
          */
         static readonly Folder: ThemeIcon;
 
-        private constructor(public id: string);
+        private constructor(public id: string, public color?: ThemeColor);
```

Comment:
I believe you are missing the optional `color` in `ThemeIcon` (which will still report as missing in the compatibility report):

```ts
/**
 * The optional ThemeColor of the icon. The color is currently only used in {@link TreeItem}.
 */
readonly color?: ThemeColor | undefined;
```

Reply:
Are you sure we need this? There is no explicit property for `id` either and it is recognized in the report. Or is this different for optional properties? In this case I'll add it. I would then also add the `id` prop to make it consistent with the VS Code API which defines both explicitly.